transforms/serializable.js

```diff
@@ -0,0 +1,60 @@
+const assemblyscript = require('assemblyscript');
+const {
+  CommonFlags,
+  Node,
+  NodeKind,
+  SourceKind,
+  TypeKind,
+  parseFile
+} = assemblyscript;
+
+
+exports.applyTransform = function(parser) {
+
+  const entrySrcIdx = parser.program.sources.findIndex(s => s.isEntry)
+  const entrySrc = parser.program.sources[entrySrcIdx]
+
+  entrySrc.statements.forEach((s, i) => {
+    if (
+      // s.kind === NodeKind.CLASSDECLARATION &&
+      s.kind === 45 && // NodeKind is not exported in the latest assemblyscript!!
```

Comment:
We should reference in this file the files where these NodeKind magic number references come from :)